### PR TITLE
feat(deployment): setting up dynamodb to dev and qa

### PR DESCRIPTION
**Pull request by BartoszZabuski**

### CODE REVIEW

File: terraform/dynamodb.tf [hunk 1]
```diff
@@ -0,0 +1,26 @@
+resource "aws_dynamodb_table" "idempotency_info" {
+
+  provider = "aws"
+
+  name           = "${var.service}-idempotency_info_${terraform.workspace}"
```
Comment (BartoszZabuski) [inline terraform/dynamodb.tf]: what do you think of content-pipeline- prefix here? or shall I remove it and keep it generic?

File: terraform/dynamodb.tf [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  read_capacity  = 3
+  write_capacity = 1
+  hash_key       = "key"
+
+  attribute {
+    name = "key"
+    type = "S"
+  }
```
Comment (ukcrpb6) [inline terraform/dynamodb.tf]: needs a seen attribute (integer)

File: terraform/dynamodb.tf [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+  provider = "aws"
+
+  name           = "${var.service}-idempotency_info_${terraform.workspace}"
+  read_capacity  = 3
+  write_capacity = 1
+  hash_key       = "key"
+
+  attribute {
+    name = "key"
```
Comment (ukcrpb6) [inline terraform/dynamodb.tf]: at present it is `id` though I can change this simply

File: terraform/dynamodb.tf (hunk 1 above)
Comment (ukcrpb6) [reply terraform/dynamodb.tf]: keep it service specific for now

**Comment by gphaydon** [comment]
is an output.tf needed to provided the details of the created dynamodb resource, so we can add to c-pipe? For instance it's endpoint?

**Comment by BartoszZabuski** [comment]
regarding outputs I'm happy to do it. I'm just not sure how can we use it in pipeline deployment? can pipeline deployment script use remote state on S3?

File: terraform/dynamodb.tf (hunk 3 above)
Comment (BartoszZabuski) [reply terraform/dynamodb.tf]: let's keep it 'id'. It's less generic

**Pull request closed without merging**